pixi.toml

```diff
@@ -0,0 +1,123 @@
+[project]
+name = "MSS"
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-64", "osx-arm64", "win-64"]
+
+[tasks]
+
+[pypi-dependencies]
+mslib = { path = ".", editable = true }
+
+[dependencies]
+basemap = ">=1.3.3"
+cftime = ">=1.0.1"
+chameleon = "*"
+defusedxml = "*"
+email_validator = "*"
+execnet = "*"
+fastkml = ">=0.11"
+flask = ">=2.3.2"
+flask-cors = "*"
+flask-httpauth = "*"
+flask-login = "*"
+flask-mail = "*"
+flask-migrate = "*"
+flask-socketio = ">=5.1.0"
+flask-sqlalchemy = ">=3.0.0"
+flask-wtf = "*"
+fs_filepicker = "*"
+git = "*"
+gitpython = "*"
+gpxpy = ">=1.4.2"
+hdf4 = "*"
+isodate = "*"
+itsdangerous = "*"
+keyring = "*"
+libtiff = "*"
+lxml = "*"
+markdown = "*"
+matplotlib = ">=3.5.3"
+metpy = "*"
+multidict = "*"
+netcdf4 = "*"
+numpy = "<2"
+owslib = ">=0.24"
+passlib = "*"
+pillow = "*"
+pint = "*"
+psycopg2 = "*"
+pycountry = "*"
+pygeoif = "<1.0.0"
+pyjwt = "*"
+pymysql = ">=0.9.3"
+pyqt = ">=5.15.0"
+pysaml2 = "*"
+python = "<3.12"
+python-engineio = ">=4"
+python-slugify = "*"
+python-socketio = ">=5"
+qt = ">=5.15.0"
+requests = ">=2.31.0"
+scipy = "*"
+shapely = ">=2.0.0"
+skyfield = ">=1.12"
+skyfield-data = ">=6"
+tk = "*"
+unicodecsv = "*"
+validate_email = "*"
+websocket-client = "*"
+werkzeug = ">=2.2.3, <3.0.0"
+xstatic = "*"
+xstatic-bootstrap = "*"
+xstatic-jquery = "*"
+
+[target.linux-64.dependencies]
+dbus-python = "*"
+libxmlsec1 = "*"
+
+[target.osx-64.dependencies]
+libxmlsec1 = "*"
+
```

Comment:
we maybe need dbus-python also on osx-64. I have this on my mss-9.3.1 installation env.

Reply:
I somehow remembered that the dbus-python package did not exist for osx at all, but it looks like it does. Weird.